yogstation/code/game/objects/items/holotool/modes.dm

```diff
@@ -0,0 +1,77 @@
+/datum/holotool_mode
+	var/name = "???"
+	var/sound
+	var/behavior
+	var/speed = 0.125
```

Comment:
The fact that this is both faster than the researchable powertools and the toolset implant, and only requires a single slot makes this seem way too strong for something that spawns in at roundstart. Especially when you consider that all researchable items are weaker than this.

Either make it top-tier research only, or severely nerf the speed to put it just below the powertools the CE starts with.